### PR TITLE
feat: Bumped SDK version to 0.50.0 in services and tools

### PR DESCRIPTION
**Pull request by chandrasekharan-zipstack**
## What

- Bumped SDK version in services and tools to `0.50.0`
- Updated SQL to return adapter name and type from `platform-service`'s `get_adapter_instance_from_db`

## Why

- SDK bump contains some changes to mark repetitive messages as DEBUG logs and few changes to log time taken by a few long running operations

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

- No, but in case of regressions in the SDK it's possible

## Related Issues or PRs

- https://github.com/Zipstack/unstract-sdk/releases/tag/v0.50.0

## Dependencies Versions

- `unstract-sdk~=0.50.0`

## Notes on Testing

- Built tool locally and checked
- Ran a WF locally

## Screenshots
- Tool run with DEBUG logs
image
- WF run with enriched adapter information
image

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

**Review by Deepak-Kesavan — APPROVED** [review]

**Review by muhammad-ali-e — APPROVED** [review]
LGTM, Hope this not break V2.

File: platform-service/src/unstract/platform_service/helper/adapter_instance.py [hunk 1]
```diff
@@ -28,21 +28,23 @@ def get_adapter_instance_from_db(
         """
         if check_feature_flag_status(FeatureFlag.MULTI_TENANCY_V2):
             query = (
-                "SELECT id, adapter_id, adapter_metadata_b FROM "
-                f'"{DB_SCHEMA}".{DBTableV2.ADAPTER_INSTANCE} x '
+                "SELECT id, adapter_id, adapter_name, adapter_type, adapter_metadata_b"
```
Comment (gaya3-zipstack) [inline platform-service/src/unstract/platform_service/helper/adapter_instance.py:31]: @chandrasekharan-zipstack  Will this cause  a change in doc_id and hence require re-indexing as we discussed the other day?

File: prompt-service/pyproject.toml [hunk 2]
```diff
@@ -15,7 +15,7 @@ dependencies = [
     "flask~=3.0",
     "llama-index==0.10.58",
     "python-dotenv==1.0.0",
-    "unstract-sdk~=0.49.0",
+    "unstract-sdk==0.49.0",
```
Comment (gaya3-zipstack) [inline prompt-service/pyproject.toml:18]: Should this be 0.50.0?

File: platform-service/src/unstract/platform_service/helper/adapter_instance.py (hunk 1 above)
Comment (chandrasekharan-zipstack) [reply platform-service/src/unstract/platform_service/helper/adapter_instance.py:31]: No @gaya3-zipstack , I made sure to pop these extra columns out to not break any existing code and just log them. If they're required elsewhere we could add an extra function / perform a migration

File: prompt-service/pyproject.toml (hunk 2 above)
Comment (chandrasekharan-zipstack) [reply prompt-service/pyproject.toml:18]: Thanks for catching this, will update

**Pull request merged**